app/policies/changelog_entry_policy.rb

```diff
@@ -1,4 +1,11 @@
 # frozen_string_literal: true
 
 class ChangelogEntryPolicy < ApplicationPolicy
+  authorize :scoped_object, optional: true
+
+  def index?
+    return true if scoped_object.present?
```

Comment:
```suggestion
    return allowed_to?(:show?, scoped_object) if scoped_object.present?
```

This is an idea, not sure if we should do it.

Source: https://actionpolicy.evilmartians.io/#/writing_policies?id=calling-other-policies

Reply:
Updated